internal/pkg/stream/ecs.go

```diff
@@ -154,6 +161,35 @@ func (s *ECSDeploymentStreamer) Fetch() (next time.Time, done bool, err error) {
 			done = true
 		}
 	}
+
+	stoppedSvcTasks, err := s.client.StoppedServiceTasks(s.cluster, s.service)
+	if err != nil {
+		if request.IsErrorThrottle(err) {
+			s.ecsRetries += 1
+			return nextFetchDate(s.clock, s.rand, s.ecsRetries), false, nil
+		}
+		return next, false, fmt.Errorf("fetch stopped tasks: %w", err)
+	}
+	s.ecsRetries = 0
+
+	var stoppedTasks []ecs.Task
+	for _, st := range stoppedSvcTasks {
+		if stoppingAt := aws.TimeValue(st.StoppingAt); stoppingAt.Before(s.deploymentCreationTime) ||
+			(strings.Contains(aws.StringValue(st.StoppedReason), ecsScalingActivity)) {
```

Comment:
Instead of filtering by time, I wonder if there is a more robust way to do this. For example, maybe we can check the `startedBy` field of each task, and see if it is `startedBy` the "PRIMARY" deployment's deployment ID 🤔 I am not sure if this is a solid solution, but I think it's worth exploring.

Reply:
This is definitely a good idea 👍

Reply:
What's the intended purpose of filtering by time here? Also, what other types of `StoppedReason` are there and why don't we care to filter those?

Reply:
For the question on 

> what other types of StoppedReasons are there

These are all some of the most common task stopped reasons that we can encounter listed in this [doc](https://repost.aws/knowledge-center/ecs-task-stopped). I am not completely aware of any other the reasons listed over here.

Reply:
> What's the intended purpose of filtering by time here?

`deploymentCreationTime` is the time when the ECS service deployment is started and `StoppingAt` gives the time when the task is transitioning from `running` state to `stopping` state. So I want to only display the tasks that are after the `deploymentCreationTime`